infra.bs

```diff
@@ -3,7 +3,7 @@ Group: WHATWG
 H1: Infra
 Shortname: infra
 Text Macro: TWITTER infrastandard
-Text Macro: LATESTRD 2023-07
+Text Macro: LATESTRD 2024-01
 Abstract: The Infra Standard aims to define the fundamental concepts upon which standards are built.
 Translation: ja https://triple-underscore.github.io/infra-ja.html
 Required IDs: tracking-vector
```

Comment:
Is this intentional? Deleting a field name but leaving the value?

Reply:
This is not being changed at all. I don't understand why it's being highlighted.